components/conference/Organization.tsx

```diff
@@ -0,0 +1,164 @@
+import * as fs from 'fs';
+import path from 'path';
+import { Col, Row } from 'react-bootstrap';
+
+import styles from './Organization.module.less';
+
+type Sponsor = {
+  level: string;
+  href: string;
+  imgSrc: string;
+};
+
+type Partner = {
+  href: string;
+  imgSrc: string;
+};
+
+export interface OrganizationInfoProps {
+  sponsors: Sponsor[];
+  partners: Partner[];
+}
+
+export async function getOrganizationInfoData(): Promise<OrganizationInfoProps> {
+  const filePath = path.join(
+    process.cwd(),
+    './components/data/SponsorData.json',
+  );
+  const text = fs.readFileSync(filePath, 'utf8');
+  let guestJSON = JSON.parse(text) as OrganizationInfoProps;
+  return guestJSON;
+}
+
+function paserSponsor(sponsor: Sponsor, index: any) {
+  return (
+    <Col className={`${styles.sponsor_item} text-center d-block`} key={index}>
+      <p className="m-0 d-block">{sponsor.level}</p>
+      <a target="_blank" href={sponsor.href} rel="noreferrer">
+        <img className="my-3" src={sponsor.imgSrc} />
+      </a>
+    </Col>
+  );
+}
+
+function OrganizationInfo(organizationInfoProps: OrganizationInfoProps) {
+  const sponsors = organizationInfoProps['sponsors'];
+  const partners = organizationInfoProps['partners'];
+
+  const platinumItem = sponsors
+    ?.filter(sponsor => sponsor.level == '铂金赞助商')
+    .map((sponsor, index) => paserSponsor(sponsor, index));
+
+  const goldItem = sponsors
+    ?.filter(sponsor => sponsor.level == '金牌赞助商')
+    .map((sponsor, index) => paserSponsor(sponsor, index));
+
+  const silverItem = sponsors
+    ?.filter(sponsor => sponsor.level == '银牌赞助商')
+    .map((sponsor, index) => paserSponsor(sponsor, index));
+
+  const copperItem = sponsors
+    ?.filter(sponsor => sponsor.level == '铜牌赞助商')
+    .map((sponsor, index) => paserSponsor(sponsor, index));
+
+  const giftItem = sponsors
+    ?.filter(sponsor => sponsor.level == '礼品赞助')
+    .map((sponsor, index) => paserSponsor(sponsor, index));
```

Comment:
```suggestion
  const {
    铂金赞助商: platinumItem,
  } = groupBy(sponsers, 'level');
```
https://web-cell.dev/web-utility/functions/groupBy.html